php-checks/src/main/java/org/sonar/php/checks/TooManyMethodsInClassCheck.java

```diff
@@ -90,6 +93,39 @@ private int getNumberOfMethods(ClassTree tree) {
     return nbMethod;
   }
 
+  private static boolean classIsDBEntity(ClassTree tree) {
+    boolean foreignMethodFound = false;
+
+    for (ClassMemberTree classMember : tree.members()) {
+      if (classMember.is(Kind.METHOD_DECLARATION)) {
+        String showMember = ((MethodDeclarationTree) classMember).name().text();
```

Comment:
When checking text, you should check case-insensitively to not miss a get like `GetFoo`